core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -639,7 +639,6 @@ private[spark] class ExternalSorter[K, V, C](
    * called by the SortShuffleWriter.
    *
    * @param blockId block ID to write to. The index file will be blockId.name + ".index".
-   * @param context a TaskContext for a running Spark task, for us to update shuffle metrics.
```

Comment:
this got removed in [SPARK-10984](https://github.com/apache/spark/commit/85e654c5ec87e666a8845bfd77185c1ea57b268a), just cleaning up the comment